src/lib/components/AssistantSettings.svelte

```diff
@@ -53,8 +56,48 @@
 	}
 
 	let loading = false;
+	let generatingAvatar = false;
 
-	let generateAvatar = false;
+	async function onGenerate() {
+		generatingAvatar = true;
+		form = null;
+
+		const generatedAvatar = await fetch(base + "/generate", {
```

Comment:
`/generate` can be a confusing route (for a text-genreation app)? maybe `generate-avatar` instead?

Reply:
fixed!